data/json/items/comestibles/carnivore.json

```diff
@@ -124,6 +124,20 @@
     }
   },
   {
+    "id": "offal_cooked",
+    "copy-from": "meat",
+    "type": "COMESTIBLE",
+    "name": "cooked offal",
+    "name_plural": "cooked offal",
+    "description": "Freshly cooked entrails.  Unappetizing, but filled with essential vitamins.",
+    "relative": {
+        "vitamins": [ [ "vitA", 5 ] ]
```

Comment:
It actually requires a couple less lines to copy from ````meat```` - no need to change calories or parasites etc.